packer/install-dependencies.sh

```diff
@@ -383,6 +383,11 @@ done
 
 df -h
 
+# warn at 75% space taken
+[ -s $(df -P | grep '/dev/sda1' | awk '0+$5 >= 30 {print}') ] || echo "WARNING: 75% of /dev/sda1 is used" >> ${RELEASE_NOTES_FILEPATH}
```

Comment:
put this to 30 on purpose so I could test it works as expected, will change to 75 after I get a signal from the VHD pipeline.